### PR TITLE
Add readme file

### PR DESCRIPTION
**Pull request by KarimAsaad798**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+![Movie-app](./01.PNG)
+
+# Movie App
+This is an awesome movie app
```
Comment (abdulrahman19) [inline README.md:4]: You forget the `.` and the end

File: README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+# Requirements
+* Angular
+* Bootstrap 4
```
Comment (abdulrahman19) [inline README.md:8]: Is this a new version?

Comment (KarimAsaad798) [reply README.md:8]: Yes this is the newest version.

Comment (abdulrahman19) [reply README.md:8]: Thanks for help

**Pull request merged**